packages/kotti-ui/source/kotti-field-select/constants.ts

```diff
@@ -31,6 +31,18 @@ export const KOTTI_FIELD_SELECT_SUPPORTS: KottiField.Supports = {
 
 export const KOTTI_FIELD_SINGLE_SELECT_PROPS = {
 	...KOTTI_FIELD_SELECT_PROPS,
```

Comment:
if you intend to extend this to `KtFieldMultiSelect` as well, then move the prop validator and defintion to `KOTTI_FIELD_SELECT_PROPS`, it should already work, and you can in theory use the same template code
you would need a sub-component to render in both that contains the template code and the styles

Reply:
I’d say this should definitely be extended to `KtFieldMultiSelect` as well. Will be really confusing if it’s not working when someone tries this

As far as I can see, this should be trivial to support for both at the same time

@RueRivoli do you want to do this or should I take over?